src/components/subject-icons/SubjectIcon.jsx

```diff
@@ -82,11 +82,11 @@ const SIZE = {
   NORMAL: 'normal'
 };
 
-const SubjectIcon = ({type, size = SIZE.NORMAL}) => {
+const SubjectIcon = ({type, size = SIZE.NORMAL, mono}) => {
   const iconClass = classNames('sg-subject-icon', {
     [`sg-subject-icon--${size}`]: size !== SIZE.NORMAL
   });
-  const iconType = `#icon-subject-${type}`;
+  const iconType = mono ? `#icon-subject-mono-${type}` : `#icon-subject-${type}`;
```

Comment:
`#icon-subject-${mono ? 'mono-' : ''}-${type}` 🤔 not sure if it looks better, just a proposition